Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
-        int digitCount = content.Count(char.IsDigit);   
+        var line = reader.ReadLine()!;
 
-        Console.WriteLine($"The file contains {letterCount} letters and {digitCount} digits.");
-    }
-    else
-    {
-        Console.WriteLine($"File not found at path: {filePath}");
-    }
+        foreach (var character in line)
+        {
+            var helper = new Helper();
+
```

Comment:
in loop it is not recommended to create new object

Reply:
Why though ?

Reply:
optimization of memory, cpu usage and you will always get initialised results of counting

Reply:
Do i need helper class at all then ?

Reply:
no, extending types would be better

Reply:
for this simple task it would be better to write in same file methods, like "IncreaseLetter_CountIfTrue"